### DISCUSSION TITLE
Remove padding from Kitty while using Yazi

### DISCUSSION THREAD

**Discussion by vdawg-git** [sxyazi/yazi#1844] [1. Q&A] [answered]
### What system are you running Yazi on?

Linux Wayland

### What terminal are you running Yazi in?

Kitty

### Describe the question

Having padding for the terminal shell is nice. For a TUI program like Yazi
not so much.

To get around this I found this snippet for Neovim:
```lua
vim.cmd([[
augroup kitty_mp
    autocmd!
    au VimLeave * if !empty($KITTY_WINDOW_ID) | :silent !kitty @ set-spacing padding=24 margin=0
    au VimEnter * if !empty($KITTY_WINDOW_ID) | :silent !kitty @ set-spacing padding=0 margin=0
    au BufEnter * if !empty($KITTY_WINDOW_ID) | :silent !kitty @ set-spacing padding=0 margin=0
augroup END
]])
```

Is there a way to have this with the `init.lua` in Yazi?

### Validations

- [X] I have searched the existing discussions/issues

**Comment by sxyazi** [comment, marked as answer]
If you're using the [Shell wrapper](https://yazi-rs.github.io/docs/quick-start#shell-wrapper) I think you can just add it to the wrapper:

```diff
@@ -1,6 +1,8 @@
 function y() {
 	local tmp="$(mktemp -t "yazi-cwd.XXXXXX")" cwd
+	kitty @ set-spacing padding=0 margin=0
 	yazi "$@" --cwd-file="$tmp"
+	kitty @ set-spacing padding=24 margin=0
 	if cwd="$(command cat -- "$tmp")" && [ -n "$cwd" ] && [ "$cwd" != "$PWD" ]; then
 		builtin cd -- "$cwd"
 	fi
```

**Reply by vdawg-git** [reply]
Perfect answer, thank you
